hack/kubearchive-install.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# Copyright KubeArchive Authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Install KubeArchive by itself. Used by quick install and also to update existing
+# KubeArchive installations.
+
+set -o errexit
+set -o xtrace
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd ${SCRIPT_DIR}/..
+
+PODS=$(kubectl -n kubearchive get pods | grep -E -v "NAME|No resources|apiserversource" |& awk '{print $1}')
+
+YAML=$(mktemp --suffix=.yaml -t kubearchive-XXX)
+bash cmd/operator/generate.sh
+helm template kubearchive charts/kubearchive -n kubearchive \
+    --include-crds \
+    --set "global.production=true" > /tmp/kubearchive-not-resolved.yaml
+ko resolve -f /tmp/kubearchive-not-resolved.yaml --base-import-paths > ${YAML}
+kubectl apply -n kubearchive -f ${YAML}
+
+rm -f ${YAML}
+
+kubectl -n kubearchive rollout status deployment --timeout=90s
+
+# Wait for all the existing pods to terminate.
+for pod in ${PODS}; do
+    kubectl -n kubearchive wait pod $pod --for=delete --timeout=90s
+done
+
+# Now make sure all the new pods are ready.
```

Comment:
Since the script will be waiting here, you should have an echo so anyone watching knows what is happening.

```suggestion
echo "Waiting for all the new pods to start up."
```

Reply:
The command is echoed here so the user will know what is going on, and `kubectl` prints the status.

```
+ kubectl -n kubearchive wait pod --all --for=condition=ready --timeout=90s
pod/kubearchive-api-server-759b549d69-wtxds condition met
pod/kubearchive-operator-6f7b5c8cd9-ghg2h condition met
pod/kubearchive-sink-66dc8fdb68-zvdlb condition met
```

Reply:
OKAY